Makefile

```diff
@@ -32,6 +32,7 @@ default:
 	@echo "shell            - open a shell in the base container"
 	@echo "clean            - remove all build, test, coverage and Python artifacts"
 	@echo "lint             - check style with flake8"
+	@echo "black            - run black on black-formatted Python files"
```

Comment:
I was thinking about these two make rules. It'd be nice to be the same across projects. It'd be nice to take into account that we're linting and fixing multiple languages. Tecken has "lintcheck" and "lintfix".

`mach` uses `mach lint` and `mach lint --fix`. We could do something like that, but it's tricky to pass arguments with make rules, so we'd still need a separate make rule.

I think I'm going to change this to "lint" and "lintfix".